org.eclipse.sprotty/src/main/java/org/eclipse/sprotty/DefaultDiagramServer.java

```diff
@@ -237,7 +237,7 @@ public void setNeedsClientLayout(boolean value) {
 	 * with the layout engine configured with {@link #setLayoutEngine(ILayoutEngine)}, so returning {@code true}
 	 * here makes sense only if such an engine is available.
 	 * 
-	 * <p>The default implementation returns the value configured with {@link #setNeedsServerLayout(boolean)},
+	 * <p>The default implementation returns the value configured with {@link #getServerLayoutKind(ServerLayoutKind)},
```

Comment:
This should point to `setServerLayoutKind`.

Reply:
Fixed